src/services/amplify.ts

```diff
@@ -4,29 +4,22 @@ import {ISensorData} from 'types/subscriptionTypes';
 import {rider} from 'graphql/subscriptions';
```

Comment:
I would take another pass at this file and perhaps merge master? More recent changes I've made are being deleted